assets/svg/bagpack.svg

```diff
@@ -0,0 +1,21 @@
74r8Y9efVes2IP3/T5x4Yyxy+Yv3jaA6UAd3reo4m4wpAQZQAAcD4UQZUHAGgIEqAAGD8KAMqjgBQFCVAADB2lAEVRwAoynIZCgDGjQK24ggARXEFAADGLhL73qIIAIVRAgQA40YZUHEEgOL4EgLAuFEGVBgBoABKgADAGcqACiIAFGAVc/YPAI4cKANCTgSAAigBAgB3KAMqhgBQACVAAOAOZUDFEACK4BFAAHCGMqBiCABFUAIEAM5QBlQMAaAIrgAAgDOUARVDACiEEiAAcIUyoGIIAMXw5QMAVygDKoQAkBMlQADgHGVABRAAcqIECADcowwoPwJATpQAAYB7lAHlRwDIiRIgAHCPMqD8CAB58QggADhHGVB+BIC8KAECAOcoA8qPAJAXVwAAwDnKgPIjAOTGFQAAcI2fAPIjAORH6gQA9wgAOREAcjhQArTU9RwAAMqA8iIA5EAJEADUB2VA+RAAcqAECADqgzKgfAgAOVACBAD1QRlQPgSAPHgEEABqgycB8iEA5EEJqJKpxgAABx1JREFUEADUBmVA+bRcD9BIxpwgWddTBOFNn3tg0T9/3De+PaZJjuxnTz/F6fJ9x/pHGpQB5cMVgFy4AgAAdcFPAPkQAPIhbQJAfRAAciAAZEQJEADUDmVAORAAMoqGEUkTAGpmfiph35wRASAjyyOAAFA/A/bNWREAMqIECADqhzKg7HgMMCseAayVE1dNOl3+z5wuHax/PIwnAbIjAGRl7WoZ10PgYdMdLmKFjPWPh1EGlB1bT1bcAwAAtUMZUHYEgMwoAQKAuuEngOwIANmRMgGgfggAGREAMqAECABqizKgjAgAGVACBAD1RRlQNgSADCgBAoAaowwoEwJABonlNyYAqCvDTdqZEAAyMDLHu54BB/velLuf/L4/ye0grrH+cRCu0mZCAMjCki7r5keTS5wcBL4/uVQ/mOIA4BrrH49FGVA2BIAsSJe1NO6DADv/emH942GUAWVDAMjEHut6AhzZuA4C7PzrifUPSbIS++gMCADZrHA9AI6u6oMAO/96Y/1DxtADkAEBIBu+XDVX1UGAnX8zsP4DZ+1y1yM0CQEgG94D3ABlHwTY+TcL6z9oQ9cDNAkBIJu9rgdAOmUdBNj5NxPrP1DGsI/OgACQhbEPuh4B6RU9CLDzbzbWf4Cs3eN6hCYhAGRgrbnP9QzIJu9BgJ2/H1j/wfmu6wGahACQgbH6V9czILusBwF2/n5h/Qflm64HaBICQAZW9h7XMyCftAcBdv5+Yv2HwRj7ddczNAkBIIN21Pq0eBKgsUYdBNj5+4317z077A3vcj1EkxAAMrhhx+33y+pe13Mgv6MdBNj5h4H177Wvv+8vPv2A6yGahACQlTG3uh4BxRx6EGDnHxbWv7fYN2fUcj1A00Qt/U0y0B+L8NRoP5pcovjArzk/4LWuwWH9eydRPPyA6yGaxrgeoIku3fiiT0j2ha7nyMomVvPz/Ux/Z3KqU9E0AMbBSpqf7WX6OxNTnUYdHKzs7TfvvPPFrudoGs5ic7BJ8l9czwAA2M/YaKvrGZqIAJDDze++8/PW6JOu5wCA0BnpEzddd8fnXM/RRASAnExi3ypp3vUcABCwBdnoCtdDNBUBIKebrrvzm5K51vUcABAu+2c3XreLR7NzIgAU8MslS6+R9AXXcwBAgD6vPT/f7nqIJiMAFHDLzC29lokvkuyPXc8CAAG5Pxq2X3HTTV/J9lgTDkIAKOiGHbffb230Mkm8hhIAqrfHWvPS917/cU68CiIAlODm6+74kpF5kQgBAFClPUkSnXfzdXd8yfUgPiAAlOTGnXd84UA50E9dzwIAHvqJtebc97171/92PYgvCAAlumnnnV8cRMkZkr7oehYA8MhXFQ+fx5l/uWLXA/jma//rvodO/5UVHzDdqX2SzlSd/o2tNBgkmf5Ku12f8QHkM+gPM/33rXZclyrgvpV2TixEl/zVu+/8hethfFOTdeynN11x3jOs0fUyOtv1LBLvAgBC1Nh3ARh9OrL2D9+7885/cT2Kr5yv4xBcuvm8s22iPzHSb8jhvzkBAAhPwwKAldVdkTH/9b0777jLzQjhIACM0e9fccGpxiSvl7EXSTp13MsnAADhaUgA+JaRvU3Gvv/GHZ/89ngXHS4CgCNvePsFT46iwTmRzK9aa54m2ZMkrZS0VFK3imUSAIDw1CgALEh6SNIvJfM9yd5rpHsGNv7MX193+/fLXxxGIQBgUZduPG9BEikAaJ6Fm3Z+YsL1EKgvHgPEKJQbAc30oOsBUG8EAIzyPdcDAMjOGHuf6xlQbwQALMpI/+x6BgDZWav/63oG1BsBAItKZD/jegYAOZjo065HQL0RALCo9mzv45L2uZ4DQCb7Wvvmd7keAvVGAMCibrjhrr1W9u9czwEgPSN95IYb7trreg7UGwEAI0XGXispW4EAAFd60XC4zfUQqD/e9IKRvvL5+35x+vPXLZXMf3Q9C4ARrNlx47vvvNX1GKg/rgAgld6S3p8YY3kPN1BvX+j2zIzrIdAMNAEitcs3XfDEgR3+H0mrXc8C4DA/iYbt57z3+o//2PUgaAauACC1G3bcfn8i89uSfuR6FgAH+aGG8Ys4+CMLAgAyed/OO+7pt6NnSfqs61kASJK+0DLxc266/vavux4EzcJNgMjsnz/3ndnnn37qR5O2TSSdIV4WBLjQM7Lbuwvx6//79bvo/Udm3AOAQg7cF/Ankl4radr1PEAA9smYv40Hg2v/6vpP0feP3AgAKMXll5+1ZDjdebG1Olsyz5R0kqQV4uoAUERP0m5J37NGX5PVP7VnF26n5AcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIzL/wfSYoHmeBnpmQAAAABJRU5ErkJggg=="/>
```

Comment:
@adi-uchiha any progress on this PR?

I was thinking if you are unable to convert these PNGs to "pure" SVGs, then its okay, it can be re-assigned if you wish.